### ISSUE TITLE
Question about mini-batches

### ISSUE THREAD

**Issue by gaklein2007** [facebookresearch/Detectron#599]
I have a question regarding the predecessor to this paper, the Faster R-CNN paper, and I thought this would be the easiest way to do this. In the paper, specifically section 3.1.3, it is mentioned that the RPN is trained by using a 1:1 ratio of positive and negatively labelled anchors. I have been looking at many different implementations of this code (specially the Mask R-CNN) and was wondering if these were positive and negative anchors based on the ground truth or predictions. Mainly, in the loss function am I evaluating all predicted anchor deltas (after non-max supresion) to a subset of ground-truth anchor pairs?
